plugins/services/src/js/service-configuration/PodHealthChecksConfigSection.js

```diff
@@ -161,13 +168,19 @@ class PodHealthChecksConfigSection extends React.Component {
               Command Health Checks
             </ConfigurationMapHeading>
             <ConfigurationMapSection key="pod-general-section">
-              <ConfigurationMapTable
-                columnDefaults={{ hideIfEmpty: true }}
-                columns={this.getCommandColumns()}
-                data={healthChecks.command}
+              <MountService.Mount
+                type="CreateService:ServiceConfigDisplay:Pod:HealthChecks:Command"
+                appConfig={this.props.appConfig}
                 onEditClick={onEditClick}
-                tabViewID="multihealthChecks"
-              />
+              >
+                <ConfigurationMapTable
+                  columnDefaults={{ hideIfEmpty: true }}
+                  columns={this.getCommandColumns()}
+                  data={healthChecks.command}
+                  onEditClick={onEditClick}
```

Comment:
@orlandohohmeier do you think it makes sense to _always_ pass `Mount` props to it's children regardless whether it's a `default` or a `registered` component? I'm talking about this https://github.com/dcos/dcos-ui/blob/master/packages/foundation-ui/src/mount/Mount.js#L78

Reply:
@nLight I wouldn't change the mount service to manage it's children.